test/functional/api/vim_spec.lua

```diff
@@ -1,6 +1,7 @@
 -- Sanity checks for vim_* API calls via msgpack-rpc
 local helpers = require('test.functional.helpers')
 local Screen = require('test.functional.ui.screen')
+local NIL = require('mpack').NIL
```

Comment:
well, it is a transitive dependency via the `nvim` module imported in helpers, which is the source of the NIL values, so I think it should be imported via `helpers`